package-lock.json

```diff
@@ -18760,7 +18760,7 @@
 				"fast-average-color": "4.3.0",
 				"lodash": "^4.17.21",
 				"memize": "^1.1.0",
-				"micromodal": "^0.4.6",
+				"micromodal": "^0.4.9",
```

Comment:
`micromodal` is still at `0.4.6` in the `package-lock.json`. The simplest way to upgrade it is to install it in the root folder and then remove it from the main `package.json` file.

Reply:
Thanks! It's updated now.